### DISCUSSION TITLE
Tenant names

### DISCUSSION THREAD

**Discussion by hishamco** [OrchardCMS/OrchardCore#10585] [General]
I'm still wondering why the tenant name don't allow spaces, it's nice to display tenant name not sort of technical name

/cc @jtkech

**Comment by jtkech** [comment]
@hishamco 

Just tried by removing the no space validation constraint, looks like it just works.

@hishamco @deanmarcussen 

But yes, before removing this validation, if it fails to validate (e.g. a space in the tenant name) when it re-displays the form in `Edit or Create.cshtml` it fails on an NRE on `Model.FeatureProfiles.Any()`, I think I know why, I will comment it in #10581.
